xapian-applications/omega/templates/query

```diff
@@ -88,7 +88,7 @@ $list{$map{$queryterms,$list{$html{$uniq{$unstem{$_}}},<b>,</b>/<b>,</b>}:&nbsp;
 <br><small>Search took $time seconds</small>
 <table>
 $hitlist{<tr><td valign=top>
-$set{clickurl,$if{$opt{clicklog},$or{$env{SCRIPT_NAME},omega.cgi}?FMT=clicklog&DB=$url{$dbname}&QUERYID=$opt{queryid}&DOCID=$url{$id}&URL=$url{$field{url}},$field{url}}}
+$set{clickurl,$if{$opt{clicklog},$or{$env{SCRIPT_NAME},omega.cgi}?FMT=clicklog&DB=$url{$dbname}&QUERYID=$opt{queryid}&DOCID=$url{$id}&URL=$url{$field{url}},$html{$field{url}}}}
```

Comment:
This undoes the (correct) fix you made two commits earlier.

As a rule of thumb, escape with `$html` at the point of use, and only there.

If you escape twice, (as happens after this change in the non-clicklog case) you'll break stuff.  For example, a URL with `&` in will get escaped to `&amp;amp;`, which the browser will unescape to `&amp;` when the result of unescaping needs to be just `&`.